scanners/headless/base.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Since the transpilation step to Node 6.10 for Lambda adds `'use strict';` to the generated files and effectively forces this onto the source files (even if we don't say `use strict` at the top of them), I'm just taking it as a push to always use `use strict` in all of our JS files and avoid any issues at transpilation time. This requires, among other things, consistently using `var` or `let` and not implicitly creating global variables.